softwareComponents/isoreconfig/include/isoreconfig/isomorphic.hpp

```diff
@@ -13,22 +13,22 @@ using Positions = std::vector< Matrix >;
 
 
 /**
- * @brief Creates a vector of positions from a given module <mod> by splitting it
- * into six distinct points, each in the center of one of the RoFIComs.
+ * @brief Creates a container of positions from a given universal module <mod> by 
```

Comment:
Remove the "universal" - it should work for any module (if not, take `UniversalModule` directly instead od `Module`.